broker/setup_broker/lsst/setup_broker.sh

```diff
@@ -118,13 +118,14 @@ manage_resources() {
             --role="${roleid}" \
             --table=true "${PROJECT_ID}:${bq_dataset}.${alerts_table}"
         gcloud pubsub subscriptions create "${subscription_bigquery_import}" \
-            --topic="${topic_bigquery_import}" \
+            --topic="${topic_alerts_json}" \
             --bigquery-table="${PROJECT_ID}:${bq_dataset}.${alerts_table}" \
             --use-table-schema \
+            --drop-unknown-fields \
             --dead-letter-topic="${deadletter_topic_bigquery_import}" \
             --max-delivery-attempts=5 \
             --dead-letter-topic-project="${PROJECT_ID}" \
-            --message-filter='attributes.schema_version = "'"${versiontag}"'"'
+            --message-filter='attributes."schema.version" = "'"${versiontag}"'"'
```

Comment:
We probably want to change these attribute names to use "_" instead of "." as the separator so that they don't require special handling.